### PR TITLE
Add information about how to set postfix permissions

### LINKED ISSUE #3591: Postfix plugin permissions
Hello,

I'm running 1.5.0 and wanted to benefit from the Postfix plugin.
How did you guys (@phemmer ?) did it with telegraf running with telegraf user which doesn't have access to files owned by postfix in /var/spool/postfix?

Example:
Error in plugin [inputs.postfix]: error scanning queue active: open /var/spool/postfix/active: permission denied

Thanks

### PR DESCRIPTION
**Pull request by danielnelson**
closes  #3591

### Required for all PRs:

- [x] Signed [CLA](https://influxdata.com/community/cla/).
- [x] Associated README.md updated.
- [ ] ~~Has appropriate unit tests.~~

### CODE REVIEW

**Comment by danielnelson** [comment]
@dynek Can you review?

**Comment by phemmer** [comment]
One caveat about these methods is that the directories in `deferred` are created on first use. So if they're created after these commands are run, then they'll need to be run again.

However in the case of ACL, you can set a default ACL, which should get inherited by the new directories (unless postfix is doing something silly like explicitly performing a `chown` instead of using a umask).

File: plugins/inputs/postfix/README.md [hunk 1]
```diff
@@ -13,6 +13,23 @@ For each of the active, hold, incoming, maildrop, and deferred queues (http://ww
...
+
+Unix permissions:
+```sh
+$ sudo chgrp -R telegraf /var/spool/postfix
+$ sudo chmod -R g+rX /var/spool/postfix
+```
+
+Posix ACL:
+```sh
+$ sudo setfacl -R -m u:telegraf:r /var/spool/postfix
```
Comment (phemmer) [inline plugins/inputs/postfix/README.md]: It'll need execute as well to be able to read the dir. `setfacl` supports `X` (upper case) to only set execute bit on dirs, not files.

**Comment by danielnelson** [comment]
I add the +X to setfacl, and the setgid bit to chmod

**Comment by dynek** [comment]
Now getting:
Dec 16 08:02:52 server.host.tld postfix/postdrop[7766]: warning: mail_queue_enter: create file maildrop/6997.7766: Permission denied

Trying to fix it :-/

**Comment by dynek** [comment]
OK fixed it with a previous dump or permissions (ls -lsaR) and `postfix check`.

Will check it a bit later.

Apparently we should not be overriding chgrping of postdrop for some dirs.

**Comment by phemmer** [comment]
yeah, looks like the `maildrop` directory does utlize group permissions. So the `chgrp` should be targetted at just the `active`, `deferred`, `hold`, `incoming`, & `maildrop` dirs.

**Comment by dynek** [comment]
Why not leave permissions as they are but play with ACL only?
```
sudo setfacl -Rdm u:telegraf:rX /var/spool/postfix/{active,hold,incoming,deferred,maildrop}
```

**Comment by danielnelson** [comment]
Updated proposed docs, can you review again?

**Pull request merged**